packages/block-library/src/image/edit.native.js

```diff
@@ -261,6 +294,56 @@ class ImageEdit extends React.Component {
 			</BottomSheet>
 		);
 
+		//Used for display of Inspector Controls
+		const getSizeSlugDisplay = ( sizeSlugValue ) => {
+			if ( sizeSlugValue === undefined ) {
+				sizeSlugValue = DEFAULT_SIZE_SLUG;
+			}
+			const sizeDisplayItems = getSizeOptionsItems();
+			switch ( sizeSlugValue ) {
+				case IMAGE_SIZE_THUMBNAIL:
+					return sizeDisplayItems[ 0 ].label;
+				case IMAGE_SIZE_MEDIUM:
+					return sizeDisplayItems[ 1 ].label;
+				case IMAGE_SIZE_FULL_SIZE:
+					return sizeDisplayItems[ 3 ].label;
+				case IMAGE_SIZE_LARGE:
+				default:
+					return sizeDisplayItems[ 2 ].label;
+			}
+		};
+
+		const getSizeOptionsItems = () => {
```

Comment:
This could be simplified if we instead define a mapping (also probably outside render):
```js
const sizeOptionLabels= {
	[ IMAGE_SIZE_THUMBNAIL ]: __( 'Thumbnail' ),
	[ IMAGE_SIZE_MEDIUM ]: __( 'Medium' ),
	[ IMAGE_SIZE_LARGE ]: __( 'Large' ),
	[ IMAGE_SIZE_FULL_SIZE ]: __( 'Full Size' ),
}
```
Then you could simply access it with `sizeOptionLabels[ sizeSlugValue || DEFAULT_SIZE_SLUG ]` instead of using `getSizeSlugDisplay`.

Reply:
Agreed this is much cleaner. Added to ImageSizePicker in 7983f04c71df03900122d95e885749c6eb1665c2